pkg/server/sandbox_status.go

```diff
@@ -69,7 +73,7 @@ func (c *criContainerdService) PodSandboxStatus(ctx context.Context, r *runtime.
 	}
 	createdAt := ctrInfo.CreatedAt
 	status := toCRISandboxStatus(sandbox.Metadata, state, createdAt, ip)
-	info, err := toCRISandboxContainerInfo(ctx, sandbox.Container, r.GetVerbose())
+	info, err := toCRISandboxInfo(ctx, sandbox, pid, processStatus, r.GetVerbose())
```

Comment:
preferred it as it was..

Reply:
It's not only "container info", right? e.g. `PodSandboxConfig`

Reply:
I meant the parameters not the name :-)